frontend/src/views/ManagePermissionsView/index.vue

```diff
@@ -215,6 +215,36 @@ const addNotifications = (newNotifications: PermissionNotificationType[]) => {
     notifications.value = newNotifications;
 };
 
+/**
+ * Handle the add permission action
+ */
+const handleAddButton = () => {
+    // FAM application id = 1
+    if (selectedApp.value?.id === 1) {
+        router.push({ name: AddFamPermissionRoute.name });
+    }
+    // App User table will have a tab index of 0
+    else if (activeTabIndex.value === 0) {
+        router.push({
+            name: AddAppPermissionRoute.name,
+            query: {
+                requestType: "addUserPermission",
+                applicationId: selectedApp.value?.id,
+            },
+        });
+    }
+    // Delegated admin table will have a tab index of 0
+    else if (activeTabIndex.value === 1) {
+        router.push({
+            name: AddAppPermissionRoute.name,
+            query: {
+                requestType: "addDelegatedAdmin",
+                applicationId: selectedApp.value?.id,
+            },
+        });
+    }
+};
```

Comment:
I guess there are not many tabs, so "0", "1" might be clear, but wondering if you like to associated the index "0" -> "App users table" and index "1" -> "Delegated admin table". such as:
```
export enum MPermissionTableTabsIndex {
  AppUsersTab,   // <-- 0 index
  DelegatedAdminTab  // <-- 1
}
```
And use the enum for the `activeTabIndex.value` state.

I guess also two `else if` can be combined as (however, separated is easier to read too):
```
    else {
        router.push({
            name: AddAppPermissionRoute.name,
            query: {
                requestType: activeTabIndex.value === 0? "addUserPermission" : "addDelegatedAdmin",
                applicationId: selectedApp.value?.id,
            },
        });
    }
```

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1671/commits/058d51c8c0bd75bf16b7653698644368ba7a1e22